internal/engine/postgresql/pg_catalog.go

```diff
@@ -6892,6 +6882,18 @@ func genPGCatalog() *catalog.Schema {
 			Args:       []*catalog.Argument{},
 			ReturnType: &ast.TypeName{Name: "uuid"},
 		},
+		{
+			Name: "generate_series",
+			Args: []*catalog.Argument{
+				{
+					Type: &ast.TypeName{Name: "numeric"},
+				},
+				{
+					Type: &ast.TypeName{Name: "numeric"},
+				},
+			},
+			ReturnType: &ast.TypeName{Name: "numeric"},
+		},
```

Comment:
Simply changed the order of the functions here to preserve the behavior of the e2e tests.